### PR TITLE
Update Report Plugin Options

### PR DESCRIPTION
**Pull request by gkahiu**
This PR contains:
- Grouping of plugin options into a tree view containing: main settings, reporting settings and scenario log settings (placeholder);

image

- Incorporation of addition reporting fields in the options widget as well as layout variables;
- Updated manual to reflect the changes in the settings UI.

This addresses the proposed changes in #321.

### CODE REVIEW

File: src/cplus_plugin/definitions/defaults.py [hunk 1]
```diff
@@ -23,7 +23,21 @@
 )
 
 OPTIONS_TITLE = "CPLUS"  # Title in the QGIS settings
+REPORT_OPTIONS_TITLE = "Reporting"
+LOG_OPTIONS_TITLE = "Logs"
 ICON_PATH = ":/plugins/cplus_plugin/icon.svg"
+REPORT_SETTINGS_ICON_PATH = str(
```
Comment (Samweli) [inline src/cplus_plugin/definitions/defaults.py:29]: Let's use one of the QGIS layouts icons with a custom CPLUS "green" color for the report settings icon.

File: src/cplus_plugin/definitions/defaults.py [hunk 2]
```diff
@@ -23,7 +23,21 @@
...
+    os.path.normpath(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+        + "/icons/report_settings.svg"
+    )
+)
+LOG_SETTINGS_ICON_PATH = str(
```
Comment (Samweli) [inline src/cplus_plugin/definitions/defaults.py:35]: Here we can use the "Result viewer" icon available under the QGIS  "Processing" menu actions as the icon for the log settings entry.

File: README.md [hunk 3]
```diff
@@ -20,9 +20,9 @@
...
-| ![GitHub Epic 1 Release)](https://img.shields.io/github/v/release/kartoza/cplus-plugin?logo=semanticrelease&label=latest-release)<br> ![](https://img.shields.io/badge/stable_version-v0.2.0dev-blue?logo=semanticrelease) | 3.0          | 3.99 |
+| Plugin version(s)                                                                                                                                                                                                          | Minimum QGIS version | Maximum QGIS version |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|------|
+| ![GitHub Epic 1 Release)](https://img.shields.io/github/v/release/kartoza/cplus-plugin?logo=semanticrelease&label=latest-release)<br> ![](https://img.shields.io/badge/stable_version-v0.2.0dev-blue?logo=semanticrelease) | 3.22                 | 3.99 |
```
[the remark this answers is not in the document]
Comment (gkahiu) [reply README.md]: Done. Separate PR [here](https://github.com/ConservationInternational/cplus-plugin/pull/371).

File: config.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "name" : "CPLUS plugin",
-    "qgisMinimumVersion": 3.0,
+    "qgisMinimumVersion": 3.22,
```
[the remark this answers is not in the document]
Comment (gkahiu) [reply config.json]: Done. See related comment.

File: config.json [hunk 5]
```diff
@@ -10,7 +10,7 @@
     "tracker": "https://github.com/kartoza/cplus-plugin/issues",
     "repository": "https://github.com/kartoza/cplus-plugin",
     "tags": ["cplus", "maps", "raster", "analytics"],
-    "category": ["plugins"],
+    "category": "Plugins",
```
[the remark this answers is not in the document]
Comment (gkahiu) [reply config.json]: Done. See related comment.

File: src/cplus_plugin/definitions/defaults.py (hunk 2 above)
Comment (gkahiu) [reply src/cplus_plugin/definitions/defaults.py:37]: Done. Adapted `processingResult.svg`.

**Review by Samweli — COMMENTED** [review]
@gkahiu looks good, after merging the other PRs there is a conflict, can you fix that then we will be good for merge, thanks

**Review by Samweli — APPROVED** [review]

**Pull request merged**